demo/.yarnrc.yml

```
@@ -0,0 +1,2 @@
+nodeLinker: node-modules
+enableGlobalCache: true 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)